waiter/config-full.edn

```diff
@@ -497,7 +518,8 @@
                                 "restart-backoff-factor" 2
                                 "scale-down-factor" 0.001
                                 "scale-factor" 1
-                                "scale-up-factor" 0.1}
+                                "scale-up-factor" 0.1
+                                "scheduler" "default"}
```

Comment:
Why do we need a default that specifies the scheduler to use is the default scheduler?